packages/common/src/seda/seda-chain.ts

```diff
@@ -455,6 +458,7 @@ export class SedaChain extends EventEmitter<EventMap> {
 			logger.error(`Transaction failed: ${result.error}`, {
 				id: txMessage.value.traceId,
 			});
+			// TODO: Discuss how do we handle this ERROR for alerting & monitoring.
```

Comment:
WARN: Even though the transaction failed. This does not need to mean that the Node is doing something wrong/there is something actionable. It could mean there is no connection, it could mean that it was just bad luck submitting.